### PR TITLE
Updated Azure Dev Ops documentation

### PR DESCRIPTION
**Pull request by abkumar01**
Updated Azure Dev Ops documentation.
1. Included the details of features added in version 2.1
2. Separated build and release documents.

### CODE REVIEW

**Comment by abkumar01** [comment]
@msnehagup @Emmanuel-Abram , please review.
@MandiOhlinger , please wait for the reviewers before merge.

**Comment by msnehagup** [comment]
You will need to update TOC.md for all the files which are new or the names have changed

File: biztalk/core/feature-pack-install-application.md
Comment (msnehagup) [inline biztalk/core/feature-pack-install-application.md:32]: Can the information for "Deployment Package Path" be called out as a Note? Right now it looks like an additional parameter

File: biztalk/core/feature-pack-create-application.md
Comment (msnehagup) [inline biztalk/core/feature-pack-create-application.md:21]: This sub-heading is redundant (for all 3 pages)

File: biztalk/core/feature-pack-create-application.md
Comment (msnehagup) [inline biztalk/core/feature-pack-create-application.md:22]: there seems to be some formatting issue here:
image

File: biztalk/core/feature-pack-update-operation.md
Comment (msnehagup) [inline biztalk/core/feature-pack-update-operation.md:21]: typo: "Create"

File: biztalk/core/feature-pack-create-application.md
Comment (msnehagup) [inline biztalk/core/feature-pack-create-application.md:31]: suggestion: use the same (or lower) heading level as Parameters

File: biztalk/core/feature-pack-release-definition.md
Comment (msnehagup) [inline biztalk/core/feature-pack-release-definition.md:50]: Suggestion:
7. Select the **Deploy** task. 
8. Select the **Operation Name** from the list and configure the associated parameters. Your options:
..


and keep the screenshot of the 3 operations here so that you don't have to repeat them on each page

File: biztalk/core/feature-pack-create-application.md
Comment (msnehagup) [reply biztalk/core/feature-pack-create-application.md:21]: OR, have a single page for all 3 operations (since they don't have that much content), and keep subheadings for each operation

File: biztalk/core/feature-pack-add-build-definitions.md
Comment (msnehagup) [inline biztalk/core/feature-pack-add-build-definitions.md:145]: does this last paragraph need to be removed since we have added a new step?

File: biztalk/core/feature-pack-create-application.md
Comment (msnehagup) [inline biztalk/core/feature-pack-create-application.md:36]: Technically, the bool is not optional since it will always have a value. My suggestion is to either remove this statement, or add it to the "Add application references" param

File: biztalk/core/feature-pack-update-operation.md
Comment (msnehagup) [inline biztalk/core/feature-pack-update-operation.md:18]: typo: Update "new"

**Review by msnehagup — CHANGES_REQUESTED** [review]

File: biztalk/core/feature-pack-create-application.md
Comment (eprofeta) [inline biztalk/core/feature-pack-create-application.md]: Deploys an application. Existing applications are uninstalled first (full stop) before new ones are installed.

File: biztalk/core/feature-pack-create-application.md
Comment (eprofeta) [inline biztalk/core/feature-pack-create-application.md]: The following parameters are optional:

File: biztalk/core/feature-pack-create-application.md
Comment (eprofeta) [inline biztalk/core/feature-pack-create-application.md]: Specifies whether the application should start once deployed. It is set to checked by default.

File: biztalk/core/feature-pack-create-application.md
Comment (eprofeta) [inline biztalk/core/feature-pack-create-application.md]: **Add referenced applications:** Comma-separated list of referenced applications, created after the deploying the main application.

File: biztalk/core/feature-pack-create-vsts-token.md
Comment (eprofeta) [inline biztalk/core/feature-pack-create-vsts-token.md:69]: BizTalk projects can be built with the Azure Pipelines agent pool instead of the Build Agent. To use the Azure Pipelines agent pool, skip this section, and go to [Step 3: Create the build and release definition](feature-pack-add-build-definitions.md).

File: biztalk/core/feature-pack-release-definition.md
Comment (eprofeta) [inline biztalk/core/feature-pack-release-definition.md]: At this point, all steps are completed. If you prefer, etc.

**Comment by abkumar01** [comment]
Created another PR with these changes: https://github.com/MicrosoftDocs/biztalk-docs/pull/827
Closing this one.

**Pull request closed without merging**
